### ISSUE TITLE
Expanding functionality of test.py

### ISSUE THREAD

**Issue by joshuahwu** [tdunnlab/scrubvae#3]
1. `train_test.py` has an issue where it overwrites previous epochs of `loss_dict.p`. Update so that if continuing training on an existing model, load in an existing `loss_dict.p` if available. If the loaded in model is different (e.g. changing parameters in some way), do not load in existing `loss_dict.p`
2. `test.py` currently only really visualizes reconstructions. Integrate `lossplot.py` for plotting training loss and test loss by epoch.
